### DISCUSSION TITLE
CertBot wildcard certificate

### DISCUSSION THREAD

**Discussion by kmccmk9** [amidaware/tacticalrmm#1090] [General]
**Server Info (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser: Firefox
 - RMM Version: latest

**Installation Method:**
  - [ ] Standard
  - [X] Docker

**Agent Info (please complete the following information):**
- Agent version: NA
- Agent OS: NA

**Describe the bug**
Following the documentation for creating certbot certificates, starting the services results in the following error messages.
<domain swapped with example.com for privacy>
[842] 2022/04/24 06:35:58.303859 [ERR] 172.16.0.3:49314 - cid:135 - TLS handshake error: remote error: tls: bad certificate
time="2022-04-24T06:38:06Z" level=fatal msg="x509: certificate is valid for *.example.com, not api..example.com". I've also tried this using the setup script in the non-docker instructions with the same result.

**To Reproduce**
1. Follow the installation instructions and generate certificates either through setup script or independently
2. Start the server and see the error.

**Expected behavior**
The server would turn on and allow connections.

**Screenshots**
NA

**Additional context**
NA

**Comment by dinger1986** [comment]
Can you share a screenshot?

I did 4 or 5 test installs last week and they worked fine every time generating new certs.

**Reply by kmccmk9** [reply]
I used certs from the setup.sh in the standalone server guide.

**Comment by wh1te909** [comment]
please paste the commands you're running to generate the certs and how you're putting them in the `.env` file, you're not doing something right

**Reply by kmccmk9** [reply]
I used certs from the setup.sh in the standalone server guide.

**Comment by ninjamonkey198206** [comment]
I run this via Docker with no issues, and have a walkthrough and example files if you need them. If you can hop on the discord and dm me, I can help you get this straightened out.

**Comment by BelleNottelling** [comment]
It looks like you added a period as a suffix to your domian name
Like this: .api.example.com
When you actually want is api.example.com

**Reply by kmccmk9** [reply]
I'm sorry where do you see that?

**Reply by BelleNottelling** [reply]
From your error message:
 certificate is valid for *.example.com, not api..example.com"

That is however, assuming you didn't introduce the typo when replacing your domain with example.com

**Reply by kmccmk9** [reply]
I'm sorry that was indeed a typo when I wrote the issue.
